cm-mlops/script/get-mlperf-inference-src/run.bat

```diff
@@ -14,13 +14,11 @@ if not exist %folder% (
        git clone %CM_GIT_RECURSE_SUBMODULES% -b "%CM_GIT_CHECKOUT%" %CM_GIT_URL% %CM_GIT_DEPTH% %folder%
        IF %ERRORLEVEL% NEQ 0 EXIT %ERRORLEVEL%
        cd %folder%
+       git checkout "%CM_GIT_SHA%"
   ) else (
-       git clone %CM_GIT_RECURSE_SUBMODULES% %CM_GIT_URL% %CM_GIT_DEPTH% %folder%
+       git clone %CM_GIT_RECURSE_SUBMODULES% -b "%CM_GIT_CHECKOUT%" %CM_GIT_URL% %CM_GIT_DEPTH% %folder%
```

Comment:
Actually we should use run.bat inside the get-git-repo script and delete this file. There, branch part is processed inside the customize.py and run.bat just needs to be made in sync with run.sh

https://github.com/mlcommons/ck/blob/master/cm-mlops/script/get-git-repo/customize.py

Reply:
Thanks @arjunsuresh, I think apart from submodule initialization and patching, basic git commands are implemented in run.bat of `get-git-repo`. Do we have need the patching part  for running the inference models?

Reply:
No. We don't need it.